app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,201 @@
+  vertical-align: baseline;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+#generalContainer {
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.pageTitle {
+  font-size: 4rem;
+  width: 100%;
+  text-align: center;
+}
+
+.userContainer {
+  height: 30vh;
+  width: 80%;
+  margin-left: 10%;
+  margin-top: 5%;
+  display: flex;
+  align-items: center;
+  border: 1px solid black;
+}
+
+.userImageContainer {
+  height: 100%;
+  width: 20%;
+  border: 1px solid black;
+}
+
+.userImage {
+  height: 100%;
+  width: 100%;
+}
+
+.userInformationContainer {
+  width: 75%;
+  margin-left: 5%;
+  height: 90%;
+}
+
+.userInformationContainer > h1 {
+  font-size: 3rem;
+  height: 80%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding-left: 5%;
+}
+
+.userInformationContainer > p {
+  font-size: 1.5rem;
+  height: 20%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding-right: 5%;
+}
+
+.bioContainer {
+  height: 40vh;
+  border: 1px solid black;
+  width: 80%;
+  margin-left: 10%;
+  margin-top: 2%;
+}
+
+.bioContainer > h2 {
+  font-size: 3rem;
+  height: 20%;
+  margin-left: 5%;
+  display: flex;
+  align-items: center;
+}
+
+.bioContainer > p {
+  font-size: 1.5rem;
+  height: 80%;
+  margin-left: 5%;
+  display: flex;
+  align-items: center;
+}
+
+.postContainer {
+  border: 1px solid black;
+  height: 25vh;
+  width: 80%;
+  margin: 2% 10%;
+}
+
+.postTitle {
+  font-size: 2rem;
+  height: 30%;
+  margin-left: 5%;
+  display: flex;
+  align-items: center;
+}
+
+.postText {
+  font-size: 1.5rem;
+  height: 40%;
+  margin-left: 5%;
+  display: flex;
+  align-items: center;
+}
+
+.postInteractionContainer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: 5%;
+}
+
+.postInteractionContainer > p {
+  font-size: 1.5rem;
+  margin: 2%;
+}
+
+.button {
+  width: 20%;
+  margin-left: 40%;
+  height: 50px;
+  border: 2px solid black;
+  background: none;
+  transition: 0.5s all ease-in-out;
+  margin-top: 200px;
+}
+
+.button:hover {
+  cursor: pointer;
+  transform: scale(1.1);
+}
+
+.indexCommentsContainer {
+  border: 1px solid black;
+  width: 80%;
+  margin-left: 10%;
+  margin-top: -2%;
+}
+
+.commentContainer {
+  width: 100%;
+  height: 40px;
+  display: flex;
+  padding: 2% 5%;
+  font-size: 1.5rem;
+}
+
+.commentInfo {
+  width: 20%;
+  font-weight: bold;
+}
+
+.commentText {
+  width: 70%;
+}
+
+.postShowContainer {
+  border: 1px solid black;
+  width: 80%;
+  margin-left: 10%;
+}
+
+.postShowInfo {
+  border: 1px solid black;
+  padding: 2% 0;
+}
+
+.mainInfo {
+  height: 10vh;
+  display: flex;
+}
+
+.mainInfo > h3 {
+  font-size: 2rem;
+  margin-left: 5%;
+  display: flex;
+  align-items: center;
+  width: 20%;
+  font-weight: bold;
+}
+
+.mainInfo > p {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 5%;
+  width: 80%;
+  margin-top: 12px;
+  font-size: 1rem;
+}
```

Comment:
- [ ] Awesome job! Your views are working great! However, let's add a few tweaks to ensure that the design complies with the wireframe provided; Let's ensure that the user picture is not joined in one border with the user bio, and similar on the other pages.

**Your designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/carreraprogrammer/rails-blog/assets/12745474/b3c396f4-acd7-4285-9d18-f0e10e11b418"> |  <img width="800" alt="Home Page" src="https://github.com/carreraprogrammer/rails-blog/assets/12745474/570d1ae2-400e-4ae1-97fa-5dd5daed8ae5"> | <img width="800" alt="Home Page" src="https://github.com/carreraprogrammer/rails-blog/assets/12745474/6293f42f-70da-4471-a063-ce35f7069ab1">

**Wireframe designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/shasherazi/blog/assets/12745474/a9461f3b-2109-4ba6-ac56-078df13764d3"> |  <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/3f7d0f65-2c5d-422d-a93c-92b8ab262892"> | <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/ff73b457-3699-46e0-9e40-a0de68408c27">